swiggy-pdp-cart/src/components/Footer/Socials/Socials.js

```diff
@@ -0,0 +1,18 @@
+import facebook from "../../../images/facebook.png";
```

Comment:
The structure has to be consistent at root as well as package level.

Now, here you have chosen to have a components folder and an images and services folder.

So, inside any component - if there are subcomponents they should go inside components folder, and if there are any images, they would go in images folder and so on

Move Socials inside components folder. Also keep folder names lowercased (read: case insensitive) so that they work the same on all environments without hiccups

Example:
`components/footer/components/socials/Socials.js`